plugins/wallabag.koplugin/main.lua

```diff
@@ -695,6 +787,37 @@ function Wallabag:setFilterTag(touchmenu_instance)
     self.tag_dialog:onShowKeyboard()
 end
 
+function Wallabag:setIgnoreTags(touchmenu_instance)
+   self.ignore_tags_dialog = InputDialog:new {
+        title =  _("Tags to ignore"),
+        description = _("Set a comma-separated list of tags to ignore"),
```

Comment:
```suggestion
        description = _("Enter one or more tags to ignore, separated by a comma."),
```

That's how it's phrased in Wikipedia languages. That being said, I like the brevity of:

```suggestion
        description = _("Enter a comma-separated list of tags to ignore."),
```

Any thoughts on consistency/style @poire-z?

Reply:
Not really :) Feel free to rewrite it the way you prefer.
About commas vs spaces as separator, some people may already have these Wikipedia lang codes separated by space - so if we go with commas, there would need so code to support both - but well... :|
If tags here should go with spaces, dunno if a tag is allowed to have some space in it - so I can't really say.
But I don't really mind about the inconsistency of the separator - as long as the text says what's to be used.

Reply:
Tags can (probably?) include a space. I was just talking about the `Enter one or more tags to ignore, separated by a space/comma.` vs `Enter a comma/space-separated list of tags to ignore.` ;-)

Reply:
Well, the 2nd is the nicest. But the first is probably the most explicite for technical and non technical people (One could wonder how to do a list of one tag: `[mytag]` `{"mytag"}`...)

Reply:
Your comments made me realize that tags can have spaces in them, so I fixed that :)
I went with the second string!